### PR TITLE
Changed to Python3 Standard

### PR DESCRIPTION
**Pull request by pypypraful**
The file is changed to python3 standard.

### CODE REVIEW

**Comment by pypypraful** [comment]
Task of putting code in def function completed.

**Review by sanjaypradeep — APPROVED** [review]
Changes are fine! Approving.

File: Collections/WordOrder.py [hunk 1]
```diff
@@ -48,16 +48,18 @@
...
-    if derp == None:
-        continue
-    else:
-        print(derp,end=' ') # comma stops print from ending with newline
+    for word in words:
+        derp = counts.pop(word, None)
+        if derp == None:
+            continue
+        else:
+            print(derp,end=' ') # comma stops print from ending with newline
```
Comment (sanjaypradeep) [inline Collections/WordOrder.py:63]: Please remove the comment.

File: Collections/WordOrder.py [hunk 2]
```diff
@@ -48,16 +48,18 @@
 
 # Enter your code here. Read input from STDIN. Print output to STDOUT
```
Comment (sanjaypradeep) [inline Collections/WordOrder.py:49]: Can remove this comment too.

**Pull request merged**